e2e/tests/recycled_storage.json

```diff
@@ -6,11 +6,11 @@
       "localStorage": [
         {
           "name": "tcHistory",
-          "value": "{\"utc\":[{\"start\":1651513945533,\"end\":1651515745533}]}"
+          "value": "{\"utc\":[{\"start\":1652301954635,\"end\":1652303754635}]}"
```

Comment:
note, this file will be regenerated each time the tests are run but do not necessary need to be checked in. I'm leaving them out of the gitignore since we will likely start doing some sort of deep inspection/modification of this file